keras_cv/models/object_detection/yolov8/__internal__/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 The KerasCV Authors
```

Comment:
We can stop using the __internal__ structure --- you can just place this in `keras_cv/models/object_detection/yolov8`

we have a change coming in that will make our API generated instead of dictated by file structure, so this is no longer needed.  Thanks!